torchci/lib/fetchPR.ts

```diff
@@ -20,11 +28,46 @@ export default async function fetchPR(
       pull_number: parseInt(prNumber),
       per_page: 100,
     }),
+    await rocksetClient.queryLambdas.executeQueryLambda(
```

Comment:
Why do you need explicit `await` here? Wouldn't `await Promise.all` do it for you?
```suggestion
    rocksetClient.queryLambdas.executeQueryLambda(
```